www/js/coolhole-util.js

```diff
@@ -4,6 +4,26 @@
  * In general, we need to keep coolhole features out of cytube files as much as possible so the future merge conflicts are kept to a minimum.
  */
 
+/**
+ * ============================================================
+ * Globals
+ * ============================================================
+ */
+const synth = window.speechSynthesis;
+let voices = [];
+function populateVoiceList() {
+  voices = synth.getVoices();
+  // If we ever wanna give users a UI for what voices are available, they can select it here
+  // Probably won't translate between different browsers but w/e
+}
```

Comment:
fix: in chrome, there's an issue where voices aren't populated on load. to fix this, we need to clear and populate the dropdown every time the voices change:
 ```suggestion
const SEC_MSG_INPUT = $("#secretaryOption-message");
const SEC_PITCH_INPUT = $("#secretaryOption-pitch");
const SEC_RATE_INPUT = $("#secretaryOption-rate");
const SEC_VOICE_INPUT = $("#secretaryOption-voice");

const synth = window.speechSynthesis;
let voices = [];
function populateVoiceList() {
  voices = synth.getVoices();
  // If we ever wanna give users a UI for what voices are available, they can select it here
  // Probably won't translate between different browsers but w/e
  
  // Clear out the secretary voices select
  SEC_VOICE_INPUT.empty();
  // Add the voices to the secretary voices select
  voices.forEach((voice) => {
    $("<option/>").val(voice.name).text(voice.name).appendTo(SEC_VOICE_INPUT);
  });
}
```

Reply:
its good